packages/vitest/src/node/logger.ts

```diff
@@ -163,8 +163,7 @@ export class Logger {
     }
     this.ctx.projects.forEach((project) => {
       const config = project.config
-      const name = project.getName()
-      const output = project.isCore() || !name ? '' : `[${name}]`
+      const output = project.isRootProject() || !project.name ? '' : `[${project.name}]`
```

Comment:
Does this require brackets?

```suggestion
      const output = (project.isRootProject() || !project.name) ? '' : `[${project.name}]`
```